src/detect.h

```diff
@@ -797,6 +800,7 @@ typedef struct SigFileLoaderStat_ {
     int total_files;
     int good_sigs_total;
     int bad_sigs_total;
+    int skipped_sigs_total;
```

Comment:
IIRC these stats can be polled over unix socket? Should we extend that support?

Reply:
can be done if follow up patch if needed

Reply:
Appears to be in the JSON stat log (ie: rules_failed) that is then exportable over the socket.

Added https://redmine.openinfosecfoundation.org/issues/6637 and will be part of the backport.